src/animation/drivers.ts

```diff
@@ -25,7 +25,7 @@ export function buildAnimation(
       return delay(options.delay ?? 0);
     default:
       console.warn(`Unsupported animation type: ${type}`);
-      return { start() {}, pause() {}, resume() {}, cancel() {}, reset() {} };
+      return { start() { }, pause() { }, resume() { }, cancel() { }, reset() { } };
```

Comment:
Currently our prettierrc config is not configured well, so It might not format correctly. There should not be changes here.